carAPI/car.log

```diff
@@ -133,3 +133,45 @@
 [2021-04-14 09:46:57] [ INFO ] [CarResource:70] Creating new car
```

Comment:
.gitignore still didnt work for this file, try to fix or dont add this file to the commit.